conans/client/build/autotools_environment.py

```diff
@@ -49,6 +50,7 @@ def __init__(self, conanfile, win_bash=False, include_rpath_flags=False):
         # Set the generic objects before mapping to env vars to let the user
         # alter some value
         self.libs = copy.copy(self._deps_cpp_info.libs)
+        self.libs.extend(copy.copy(self._deps_cpp_info.system_libs))
```

Comment:
It is ok to have libs mixed with system_libs in order to avoid breaking users, but I would like to have a dedicated `SYSTEM_LIBS` var added to the output of the generator so users can have the system libs separated from all the values

Reply:
LIBS is a standard autotools variable, it will get the libs to link from there, which is how the helper works. SYSTEM LIBS is not recognized https://www.gnu.org/software/autoconf/manual/autoconf-2.66/html_node/Environment-Variable-Index.html